recipes/util-linux/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "util-linux" %}
+{% set version = "2.30" %}
+{% set sha256 = "e84cc32566abf28b84efb5ee326e1f3597c3d19b9916740f50c75bd57e498ce0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.tar.gz
```

Comment:
Please use `{{ version }}` in place of all of the hard coded version numbers in the url.